content/vendor/additional-objects/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+date: "2019-11-27T00:00:00Z"
+lastmod: "2019-11-27T00:00:00Z"
+title: "Additional Kubernetes Objects"
+aliases: [/vendor/additional-objects]
+weight: "2"
+hideFromList: true
+---
+Elements of a KOTS application lifecycle are controlled by various Custom Resource Definitions included in a KOTS application spec.  
+
```

Comment:
I don’t know if we have defined a term “KOTS application spec” anywhere. What if this was changed to be “included as part of a KOTS application.”?